syntaxes/hlasm.tmLanguage.json

```diff
@@ -104,13 +128,15 @@
 			"_description": "This is for a line consisting of only an argument. This implies the previous line was a continuation",
 			"patterns": [
 				{
-					"_description": "A line matching this syntax should follow the argument syntax",
-					"match": "^\\s+(\\S+=.*)$",
+					"match": "^\\s{15}(\\S+)(\\s.*)$",
```

Comment:
This regex will not match a line that continues and has no spacing at the end of the line.

So in my program:

```
         STORAGE OBTAIN,                                               +
               LENGTH=DCBAREAL,                                        +
               LOC=24,            Must be below the line due to DCB    +
               LINKAGE=SYSTEM,    requirements.                        +
               COND=YES <----- In my code there is no space at the end of this statement
```

Suggestion for a fix would be to modify the second capture group to be `(\\s.*)?`. The ? indicates to match 0 or 1 times for the entire group.

Reply:
Also note when I do put a space after that `COND=YES` the highlighting still fails. I don't have a reason for that yet but I will see if I can find out why.

Out of curiosity are you testing out your changes with some personal assembler code to validate your changes?

Reply:
I'm using code that I downloaded from z/OS which is why I didn't notice the "short lines" issue.